frontend/resourceadm/components/AccessListMembers/AccessListMembersTable.tsx

```diff
@@ -51,12 +52,27 @@ export const AccessListMembersTable = ({
               </Table.Cell>
               <Table.Cell>
                 <StudioButton
+                  aria-label={
+                    isAdd
+                      ? t('resourceadm.listadmin_add_to_list_org', { org: item.orgName })
+                      : t('resourceadm.listadmin_remove_from_list_org', { org: item.orgName })
+                  }
                   onClick={() => onButtonClick(item)}
                   disabled={disableButtonFn ? disableButtonFn(item) : false}
                   variant='tertiary'
                   size='small'
                 >
-                  {buttonNode}
+                  {isAdd ? (
```

Comment:
Maybe this can be its own function or component? 😄

Reply:
fixed, added function to create button markup